asterix/cat_001.ksy

```diff
@@ -0,0 +1,93 @@
+meta:
+  id: cat_001
+  license: GPL-3.0-only
+  endian: be
+  imports:
+      - field_spec
+      - explicit
+      - cat_001_catalog
+  ks-version: 0.9
+
+doc: |
+  Implementantion of ASTERIX format.
+
+  From Eurocontrol Asterix web page at
+  https://www.eurocontrol.int/asterix
+
+  Category 001 definition file
+  RADAR DATA EXCHANGE Part 2a
+  Transmission of Monoradar Data Target Reports
+
+  SUR.ET1.ST05.2000-STD-02a-01
+
+  Edition : 1.2
+
+seq:
+  - id: fspec
+    type: field_spec
+
+  - id: data
+    type:
+      switch-on: is_plot
+      cases:
+        true:  uap_type(fspec, [10,20,40,70,90,130,141,50,120,131,80,100,60,30,150].as<s2[]>.as<str>)
+        false: uap_type(fspec, [10,20,161,40,42,200,70,90,141,130,131,120,170,210,50,80,100,60,30].as<s2[]>.as<str>)
+
```

Comment:
should the abgreviation be expanded?

Reply:
What do you mean with abbreviations?

Reply:
uap, isp, irfs,... Though feel free not to change if you consider that the abbreviations are better here.

Reply:
I try to keep it consistent with vocabulary in the standard. Although it could be have changes in the future. Take into account that the names are restricted to Kaitai impositions, for example I would use UAP, instead of uap, but it is not possible.